<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Проект</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>740</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>800</width>
      <height>150</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="addPointButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>650</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить точку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePointButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>680</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить точку</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="navigationData">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setMarkerButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>620</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Уст. маркер ВЫКЛ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectDroneButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>710</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключить дрон</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutProjectButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>740</y>
      <width>150</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>О проекте</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Навигационные данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>319</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Полученные изображения</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="photoData">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>350</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
